libafl_frida/src/helper.rs

```diff
@@ -25,6 +25,10 @@ use frida_gum::{
     stalker::{StalkerOutput, Transformer},
     CpuContext, ModuleDetails, ModuleMap,
```

Comment:
ah forgot to delete `CpuContext` from this line

Reply:
Can you open a fix pr?